intermediate/challenge-1/src/main/java/com/audela/challenge/busapi/entity/DriverEntity.java

```diff
@@ -0,0 +1,32 @@
+package com.audela.challenge.busapi.entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+
+/**
+ * A driver has a first name, last name, social security number and an email
+ */
+@Data
+@Table(name="DRIVER")
+@Entity
+public class DriverEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name="id",nullable = false)
+    private Integer id;
+
+    @Column(name="first_name",nullable = false, length = 70)
+    private String firstName;
+
+    @Column(name="last_name",nullable = false, length = 70)
+    private String lastName;
+
+    @Column(name="ssn",nullable = false, length = 70)
+    private String ssn;
```

Comment:
how would you handle sensitive information in a real application ?

Reply:
Sensitive information should be encrypted in reality. in some cases we can simply save the hash code of the sensitive information. but we cannot decrypt the hash. Password can be saved like this and we can validate the hash if required.